var/spack/repos/builtin/packages/openblas/package.py

```diff
@@ -539,6 +541,9 @@ def make_defs(self):
         if self.spec.satisfies("+bignuma"):
             make_defs.append("BIGNUMA=1")
 
+        if not self.spec.satisfies("target=avx512"):
```

Comment:
This is wrong, you should use `target=x86_64_v4:`

Reply:
Thanks for confirming. Done